content/docs/conditional-rendering.md

```diff
@@ -152,6 +152,19 @@ It works because in JavaScript, `true && expression` always evaluates to `expres
 
 Therefore, if the condition is `true`, the element right after `&&` will appear in the output. If it is `false`, React will ignore and skip it.
 
+Note that returning a falsy expression will still cause the element after `&&` to be skipped but will return the falsy expression. In the example below, 0 will be returned by the expression.
```

Comment:
I think we can be a bit more specific what "the expression" is and instead explain directly what is returned from render. `<div>0</div>`

Reply:
Thanks! That makes more sense.